reference/HA.md

```diff
@@ -1,7 +1,12 @@
 ## High Availability 
 
-Mayastor 2.0 brings in the High Availability (HA) feature. In the event of a failure, this feature will guarantee I/O continuity. 
-The HA feature consists of two components: the HA node agent and the cluster agent. The HA node agent looks for io-path failures from applications to their corresponding targets. If any such broken path is encountered, the HA node agent informs the cluster agent. The cluster-agent then creates a new target on a different (live) node. Once the target is created, the `csi-node` establishes a new path between the application and its corresponding target. The HA feature restores the broken path within seconds, ensuring negligible downtime. 
+Mayastor 2.0 brings in the High Availability (HA) feature. In the event of a failure, this feature attempts to ensure I/O continuity. 
+The HA feature consists of two components: the HA node agent (which runs in each csi- node) and the cluster agent (which runs alongside the agent-core). The HA node agent looks for io-path failures from applications to their corresponding targets. If any such broken path is encountered, the HA node agent informs the cluster agent. The cluster-agent then creates a new target on a different (live) node. Once the target is created, the `csi-node` establishes a new path between the application and its corresponding target. The HA feature restores the broken path within seconds, ensuring negligible downtime. 
+
+{% hint style="warning" %}
+The volume's replica count must be higher than 1 for a new target to be established as part of Switchover.
```

Comment:
Should this be switch-over instead of Switchover?

Reply:
Updated the text. Thanks.